src/app/shared/common/get-file-type.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Returns a 'type' part of the mimeType header
+ * image/jpeg => image
+ * video/ogg => video
+ */
+export function getFileType(mimeType: string): 'image' | 'video' | 'audio' | string | null {
```

Comment:
Change this function to return a human-readable representation of the mime type
eg. "application/xlsx" -> "spreadsheet" etc.
Handle non-common mime types in some sensible way which limits the length of the returned string.